mne/viz/_brain/_brain.py

```diff
@@ -1130,13 +1130,15 @@ def _configure_vertex_time_course(self):
         else:
             self.clear_glyphs()
 
-        # plot the GFP
+        # plot RMS of the activation
         y = np.concatenate(list(v[0] for v in self.act_data_smooth.values()
                                 if v[0] is not None))
-        y = np.linalg.norm(y, axis=0) / np.sqrt(len(y))
-        self.gfp, = self.mpl_canvas.axes.plot(
-            self._data['time'], y,
-            lw=3, label='GFP', zorder=3, color=self._fg_color,
+        rms = np.sqrt((y ** 2).mean(axis=0))
```

Comment:
Previous code was more efficient as did not need to allocate y**2

Reply:
Yes, but does it really matter?
Can of course revert if you prefer!